packages/bpk-component-carousel/src/utils-test.tsx

```diff
@@ -16,14 +16,17 @@
  * limitations under the License.
  */
 
-import { renderHook } from '@testing-library/react'
+import { renderHook } from '@testing-library/react';
 
 import { useScrollToInitialImage } from './utils';
 
 describe('useScrollToInitialImage', () => {
   it('should scroll to initial image on mount', () => {
+    const element = document.createElement('div');
+    element.scrollIntoView = jest.fn();
```

Comment:
This test was broken but was not being run because the test pattern specifies `-test` rather than `.test`. Renamed and fixed.